docs/release-notes/v0.5.0.md

```diff
@@ -0,0 +1,12 @@
+## Important notice
+
+* To ensure correct display of rocks/routes and a working rock/route search,
```

Comment:
Should be in German, I guess

Reply:
Normally we do all the release notes in English but this case is a bit special. German is also fine.